### ISSUE TITLE
testing-microflows-with-the-unit-testing-module

### ISSUE THREAD

**Issue by patrick-verhoeven-emixa** [mendix/docs#8144]
Please use the form below, leaving the prefilled data to help us. Thank you.

Page link: [testing-microflows-with-the-unit-testing-module](https://docs.mendix.com/refguide/testing-microflows-with-unit-testing-module/)

Document link: [testing-microflows-with-unit-testing.md](https://github.com/mendix/docs/blob/development/content/en/docs/refguide/modeling/application-logic/microflows-and-nanoflows/microflows/testing-microflows-with-unit-testing.md)

## My Issue/Suggestion
The discribed walkthrough and tests are not working when following all steps as described.

**Comment by Yiyun333** [comment]
Hi @patrick-verhoeven-emixa,

Thanks a lot for the feedback! I'll investigate this. 
If you can point out particularly which sections/steps do not work, that will be very helpful. 

Best,
Yiyun

**Comment by patrick-verhoeven-emixa** [comment]
Hi Yiyun,

After copying the microflows for each level the inital value van Level enum is not changed so in my opinion the second and third test microflow run into an error because in these microflows the level is still empty in the new created NewEmployee object.

Another question. Why is the object Employee set as a parameter in these microflows as the documentation (4.1)  describes 'A test microflow should have no input arguments'.  (https://docs.mendix.com/appstore/modules/unit-testing/)

Regards,
Patrick

**Comment by Yiyun333** [comment]
Hi Patrick,

Thanks a lot for providing the above info. I'll follow this up with the dedicated dev team and get back to you when we have the answers. 

Regards,
Yiyun

**Comment by Yiyun333** [comment]
Hi Patrick,

I've verified your feedback with the team. You are right. We've updated the how-to based on your feedback in this PR: https://github.com/mendix/docs/pull/8177. The updated version will be online the latest by the end of the day.

As for your question on why the object Employee was set as a parameter in the test microflows, this is indeed also not correct. We've removed this from the how-to. 

I am closing this issue now. Thanks again for your feedback! 

Regards,
Yiyun
